.github/workflows/release.yml

```diff
@@ -76,13 +56,21 @@ jobs:
           default: true
           override: true
 
-      - name: Build
-        run: cargo build --all --release && strip target/release/viceroy && mv target/release/viceroy target/release/viceroy_darwin-amd64
+      - name: Extract tag name
+        uses: olegtarasov/get-tag@v2.1
+        id: tagName
+
+      - name: Build and package
+        run: |
+          cargo build --all --release
```

Comment:
Could we also use `--locked` here, so that we never create release binaries that wouldn't have an up-to-date lockfile?

Reply:
Added!